app/controllers/pagerouters/EligibilityRouter.scala

```diff
@@ -54,34 +55,43 @@ object EligibilityRouter {
   /**
    * To be used when there are more than one 'flavour' of a lockout page. Design want them to have different urls.
    */
-  def whichGenericEligibilityPage(taxRegime: TaxRegime): Call = taxRegime match {
+  private def whichGenericEligibilityPage(taxRegime: TaxRegime): Call = taxRegime match {
     case TaxRegime.Epaye => routes.IneligibleController.payeGenericIneligiblePage
     case TaxRegime.Vat   => routes.IneligibleController.vatGenericIneligiblePage
   }
 
-  def whichDebtTooLargePage(taxRegime: TaxRegime): Call = taxRegime match {
+  private def whichDebtTooLargePage(taxRegime: TaxRegime): Call = taxRegime match {
     case TaxRegime.Epaye => routes.IneligibleController.epayeDebtTooLargePage
     case TaxRegime.Vat   => routes.IneligibleController.vatDebtTooLargePage
   }
 
-  def whichDebtTooSmallPage(taxRegime: TaxRegime): Call = taxRegime match {
+  private def whichDebtTooSmallPage(taxRegime: TaxRegime): Call = taxRegime match {
     case TaxRegime.Epaye => routes.IneligibleController.epayeDebtTooSmallPage
     case TaxRegime.Vat   => routes.IneligibleController.vatDebtTooSmallPage
   }
 
-  def whichExistingPlanPage(taxRegime: TaxRegime): Call = taxRegime match {
+  private def whichExistingPlanPage(taxRegime: TaxRegime): Call = taxRegime match {
     case TaxRegime.Epaye => routes.IneligibleController.epayeAlreadyHaveAPaymentPlanPage
     case TaxRegime.Vat   => routes.IneligibleController.vatAlreadyHaveAPaymentPlanPage
   }
 
-  def whichDebtTooOldPage(taxRegime: TaxRegime): Call = taxRegime match {
+  private def whichDebtTooOldPage(taxRegime: TaxRegime): Call = taxRegime match {
     case TaxRegime.Epaye => routes.IneligibleController.epayeDebtTooOldPage
     case TaxRegime.Vat   => routes.IneligibleController.vatDebtTooOldPage
   }
 
-  def whichFileYourReturnsPage(taxRegime: TaxRegime): Call = taxRegime match {
+  private def whichFileYourReturnsPage(taxRegime: TaxRegime): Call = taxRegime match {
     case TaxRegime.Epaye => routes.IneligibleController.epayeFileYourReturnPage
     case TaxRegime.Vat   => routes.IneligibleController.vatFileYourReturnPage
   }
 
+  //requirement from business that if multiple reasons exist but any of them are isLessThanMinDebtAllowance, go to debt too small page (PAYE only).
```

Comment:
I realise this method probably seems a bit weird, but I wanted to keep the match statement above with the other checks on eligibility status a bit cleaner by not including the logic to check tax type so that's why this was added